lua/neo-tree/sources/common/components.lua

```diff
@@ -327,6 +327,18 @@ M.name = function(config, node, state)
     end
   end
 
+  if config.highlight_open_files then
+    for _, buf in ipairs(vim.api.nvim_list_bufs()) do
```

Comment:
This is really the wrong place to do this, because then it will do this loop for every row in the tree which is wasteful.